apps/explorer-e2e/src/specs/address.spec.ts

```diff
@@ -75,7 +75,7 @@ test('address displays the intended data', async ({ page }) => {
   await explorerApp.goTo('/address/' + address)
   await expect(page.getByText(`Address ${address.slice(0, 5)}`)).toBeVisible()
   await expect(page.getByText(events.data[0].id.slice(0, 5))).toBeVisible()
-  await expect(page.getByText(`${events.data.length} events`)).toBeVisible()
+  expect(await page.getByTestId('entity-link').count()).toBe(events.data.length)
```

Comment:
iirc this will not wait for the expected count, it evaluates count() immediately. I think you are looking for:
```
await expect(page.getByTestId('entity-link')).toHaveCount(events.data.length)
```